applications_tests/lethe-fluid-matrix-free/cylinder_kelly_estimator.prm

```diff
@@ -108,10 +108,10 @@ subsection boundary conditions
   subsection bc 2
     set type = slip
   end
-  subsection bc 3
-    set beta = 1
-    set type = outlet
-  end
+  #subsection bc 3
+  #  set beta = 1
+  #  set type = outlet
+  #end
```

Comment:
It's needed because the matrix free method does not support outlets right now. If you don't mind I would leave it commented, I plan on adding the outlet soon and then I would revert back this test to it's original form.